content/en/cosign/openid_signing.md

```diff
@@ -4,71 +4,38 @@ category: "Cosign"
+## Root of Trust
 
-Information on the Fulcio root CA can be found in the [fulcio repository](https://github.com/sigstore/fulcio).
+Sigstore's root of trust, which includes Fulcio's root CA certificate and Rekor's public key, are distributed by The Update Framework (TUF). TUF is a framework to provide secure software and file updates. TUF defines a set of protocols to protect against [various types of attacks](https://theupdateframework.io/security/). For more information about how Sigstore uses TUF, see [How Sigstore uses TUF](https://dlorenc.medium.com/using-the-update-framework-in-sigstore-dc393cfe6b52).
 
-### Keys
+## Identity Tokens
 
-The root CA keys are hard-coded in Cosign today. They can only be changed by recompiling the binary. This will be made more configurable in the future.
+Cosign supports two OAuth flows today: the standard flow and the device flow.
 
-### OAuth flows
+When there is no terminal attached (non-interactive mode), Cosign will automatically use the device flow where a link is printed to stdout. This link must be opened in a browser to complete the flow.
 
-Cosign supports two OAuth flows today: the standard flow and the device flow.
+In automated environments, Cosign also supports directly using OIDC Identity Tokens from specific issuers. These can be supplied on the command line with the `--identity-token` flag. The `audiences` claim in the token must contain `sigstore`.
 
-When there is no terminal attached (non-interactive mode), `cosign` will automatically use the device flow where a link is printed to stdout. This link must be opened in a browser to complete the flow.
+### Logging in to identity issuers
 
-### Identity tokens
+To log in and set up your OIDC Identity, follow these steps:
 
-In automated environments, cosign also supports directly using OIDC Identity Tokens from specific issuers. These can be supplied on the command line with the `--identity-token` flag. The `audiences` field must contain `sigstore`.
+1. Follow the link that Cosign provides to the Issuer login page.
+2. Log in to one of the supported systems that is shown:
+![Identity Sign-in](/cosign_identity_login.png)
+3. Go back to your terminal or other application. Redirection back to Cosign is seamless.
 
-Cosign also has support for detecting some of these automated environments and producing an identity token. Currently this supports Google and Github.
+Currently Sigstore supports Microsoft, Google, and GitHub. As an alternative you can also use the environment variable `SIGSTORE_ID_TOKEN` to identify yourself. Cosign also has support for detecting some of these automated environments and producing an identity token. Currently this supports Google and GitHub, in addition to other environments. See [Cosign's providers](https://github.com/sigstore/cosign/tree/main/pkg/providers) for a complete list.
```

Comment:
"identify yourself" sounds a little not specific enough to me, should it also reference an identity token?

```suggestion
Currently, Sigstore supports Microsoft, Google, and GitHub. As an alternative, you can also use the environment variable `SIGSTORE_ID_TOKEN` to identify yourself. Cosign also has support for detecting some of these automated environments and producing an identity token. Currently this supports Google and GitHub, in addition to other environments. See [Cosign's providers](https://github.com/sigstore/cosign/tree/main/pkg/providers) for a complete list.
```

Reply:
SHould it be "Identify yourself with an identity token"?